CRM/Odoosync/Sync/Contact/SyncInformation.php

```diff
@@ -1,91 +1,112 @@
 <?php
 
 /**
- * Gets appropriate contact for synchronization with Odoo
+ * Updates contact sync information when sync successful/error
  */
 class CRM_Odoosync_Sync_Contact_SyncInformation {
 
   /**
-   * Sync contact id
+   * Updates contact sync information when synchronization was successful
    *
-   * @var int
+   * @param int $partnerId
+   * @param int $contactId
    */
-  private $syncStatusFieldId;
+  public function updateContactSuccessfulSync($partnerId, $contactId) {
+    $query = "
```

Comment:
I would fetch the option value using OptionValue API instead, then use it in the query instead of using joins. same thing for updateContactErrorSync method